lib/recurly/risk/three-d-secure/strategy/braintree.js

```diff
@@ -4,13 +4,41 @@ import ThreeDSecureStrategy from './strategy';
 const debug = require('debug')('recurly:risk:three-d-secure:braintree');
 
 export default class BraintreeStrategy extends ThreeDSecureStrategy {
-
   static strategyName = 'braintree_blue';
 
   loadBraintreeLibraries () {
     return BraintreeLoader.loadModules('threeDSecure');
   }
 
+  static preflight ({ recurly, number, month, year, cvv }) {
+    const { proactive } = recurly.config.risk.threeDSecure;
+
+    if(!proactive.enabled) {
+      return Promise.resolve();
+    }
+
+    const data = {
+      gatewayType: BraintreeStrategy.strategyName,
+      gatewayCode: proactive.gateway_code,
```

Comment:
Prefer camelCase in this domain, i.e. `gatewayCode`.

If this were to originate from an API response, then snake case would be expected.